packages/field-base/src/active-mixin.js

```diff
@@ -0,0 +1,78 @@
+import { dedupingMixin } from '@polymer/polymer/lib/utils/mixin.js';
+import { GestureEventListeners } from '@polymer/polymer/lib/mixins/gesture-event-listeners.js';
+import { DisabledMixin } from './disabled-mixin.js';
+
+const ActiveMixinImplementation = (superclass) =>
+  class ActiveMixinClass extends DisabledMixin(GestureEventListeners(superclass)) {
+    [ActiveMixin] = true;
+
+    /**
+     * An array of activation keys.
```

Comment:
Let's mark this as `@protected`.

Reply:
Done. Could you check also the `@return` property, does it look correct?

Reply:
Yes, looks good to me 👍